### PR TITLE
banner changes

### PR DESCRIPTION
**Pull request by vivekshankar1**
Signed-off-by: Vivek Shankar <vshankar@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
Added warning banner for Automate HA and removed 4.x warning.

### :chains: Related Resources
https://github.com/chef/chef-web-docs/pull/3980
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by shaik80 — APPROVED** [review]

**Review by vivek-yadav — APPROVED** [review]

File: components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md [hunk 1]
```diff
@@ -0,0 +1 @@
+We are currently working on making the setup and upgrade process to Automate HA a seamless experience. If you are already using Automate HA or planning to use it, please get in touch with Customer Success or your Account Manager for more information
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md:1]:
```suggestion
We are currently working on making the setup and upgrade process to Automate HA a seamless experience. If you are already using Chef Automate HA, or are planning to use it, please contact [Chef Support](https://www.chef.io/support) or your account manager for more information.
```

File: components/docs-chef-io/layouts/shortcodes/automate/4x-warn.md [hunk 2]
```diff
@@ -1 +1 @@
-Chef Automate 4.x will not be available for download before the end of September 2022. We are working on making the upgrade process a seamless experience. Until then, you can download Chef Automate 3.0.49. Please get in touch with support for more information.
+Chef Automate 4.x will not be available for download before the end of September 2022. We are working on making the upgrade process a seamless experience. Until then, you can download Chef Automate 3.0.49. Please get in touch with support for more information.
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/4x-warn.md:1]: Delete this file.

File: components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md (hunk 1 above)
Comment (vivekshankar1) [reply components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md:1]: This message has been suggested by @vivekvardwaz  and @nischalsrinivas . Please could you get in touch with them to get it approved.

File: components/docs-chef-io/layouts/shortcodes/automate/4x-warn.md (hunk 2 above)
Comment (vivekshankar1) [reply components/docs-chef-io/layouts/shortcodes/automate/4x-warn.md]: Done

File: components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md (hunk 1 above)
Comment (vivekshankar1) [reply components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md:1]: Hi @IanMadd ,
@nischalsrinivas has requested the below change:
```suggestion
We are currently working on making the setup and upgrade process to Automate HA a seamless experience. If you are already using Chef Automate HA, or are planning to use it, please contact your customer success manager or your account manager for more information.
```

Comment (vivekshankar1) [reply components/docs-chef-io/layouts/shortcodes/automate/ha-warn.md]: @IanMadd , i have done the suggested changes, please approve

**Review by IanMadd — APPROVED** [review]

**Pull request merged**